src/applications/drydock/interface/command/DrydockSSHCommandInterface.php

```diff
@@ -50,13 +50,20 @@ public function getExecFuture($command) {
       $flags[] = 'ConnectTimeout='.$this->connectTimeout;
     }
 
+    $ssh_host = $this->getConfig('host');
+
+    echo tsprintf(
```

Comment:
does tsprintf exists in php's stdlib?

Reply:
It's defined in arcanist 🙃 
https://we.phorge.it/source/arcanist/browse/master/src/xsprintf/tsprintf.php#L16